source/operations/concepts/availability-and-resiliency.rst

```diff
@@ -158,4 +161,48 @@ For multi-pool MinIO deployments, each pool requires at least one erasure set ma
    Use replicated remotes to restore the lost data to the deployment.
    All data stored on the healthy pools remain safe on disk.
 
+Replicated MinIO Deployments
+----------------------------
+
+MinIO implements :ref:`site replication <minio-site-replication-overview>` as the primary measure for ensuring Business Continuity and Disaster Recovery (BC/DR) in the case of both small and large scale data loss in a MinIO deployment.
+   .. figure:: /images/availability/availability-multi-site-setup.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment during initial setup
+
+      Each peer site is deployed to an independent datacenter to provide protection from large-scale failure or disaster.
+      If one datacenter goes completely offline, clients can fail over to the other site.
+
+MinIO replication can automatically heal a site that has partial or total data loss due to transient or sustained downtime. 
+   .. figure:: /images/availability/availability-multi-site-healing.svg
+      :figwidth: 100%
+      :alt: Diagram of a multi-site deployment while healing
+
+      Datacenter 2 was down and Site B requires resynchronization.
+      The Load Balancer handles routing operations to Site A in Datacenter 1.
+      Site A continuously replicates data to Site B.
+
+   Once all data synchronizes, you can restore normal connectivity to that site.
+   Depending on the amount of replication lag, latency between sites and overall workload :abbr:`I/O (Input / Output)`, you may need to temporarily stop write operations to allow the sites to completely catch up.
+
+   If a peer site completely fails, you can remove that site from the configuration entirely.
+   The load balancer configuration should also remove that site to avoid routing client requests to the offline site.
+
+   You can then restore the peer site, either after repairing the original hardware or replacing it entirely, by adding it back to the site replication configuration.
```

Comment:
Is this expanding the site replication? I know there were issues initially with this, but I can't remember what the end of the fixes was.

It's worth linking the last last phrase to one of the tutorials. Maybe https://min.io/docs/minio/linux/operations/install-deploy-manage/multi-site-replication.html#expand-site-replication.